### PR TITLE
ِApply course component

### PR DESCRIPTION
**Pull request by AbdallahAmmar96**
### main 
- make the apply page for the student 
so he can see the course and choose the days and the time for the course 
### minor
- create a new component that all to pick from many options 

Relates #29 #62 #26

### CODE REVIEW

**Comment by AsalaKM** [comment]
Why your PR was contain two different PRs
Why you pushed the profile files/component again in this PR

File: Client/src/components/contanier/student/Coures/index.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  margin-left: 20px;
+  margin-top: 35px;
+  letter-spacing: 1px;
+}
+
+.courses-card {
+  position: relative;
+}
+
+.courses-btn {
```
Comment (AsalaKM) [inline Client/src/components/contanier/student/Coures/index.css]: The cursor here should be pointer for the UX
`cursor:pointer;`

File: Client/src/components/contanier/student/Apply/index.js [hunk 2]
```diff
@@ -1,10 +1,29 @@
...
+  state = {
+    course: {
+      id:0,
+      titel: 'Writing & Speaking',
+      desc: 'In this course you will learn the fundemtal of speaking and writign skills'
+    }
+  }
   render() {
+    const days = ["Pick a day ..", "Sat - Mon - Wedn", "Sun - Tues - Thurs"];
+    const hours = ["Pick a time ..", "8:00 - 10:00 PM", "10:00 - 12:00 PM", "12:00 - 14:00 AM"];
```
Comment (AsalaKM) [inline Client/src/components/contanier/student/Apply/index.js]: can you put it in your state for easy read and to simplify your work when you fetch the data

**Review by AsalaKM — CHANGES_REQUESTED** [review]
For more professionalism
Unplug your tasks

**Comment by AbdallahAmmar96** [comment]
It's because I create new branches from other branches with unmerged commits.
The main reason is that I didn't want to wait for reviews as I would take a very long time.

@AsalaKM

**Comment by AsalaKM** [comment]
If You merged the profile PR, this can cause a lot of problems.
There is no need to re-push the previous in each time you pushed new task
**Split** your work please !!

File: Client/src/components/common/TextCard/index.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+.textcard-label {
+  text-align: left;
+  color: #06468F;
+}
+
+.textcard-data {
+  text-align: center;
+  color: #5F5F5F;
+  padding: 5px;
+}
```
Comment (Fatmasiam) [inline Client/src/components/common/TextCard/index.css]: Add new line here ?!

File: Client/src/components/contanier/student/Profile/index.css [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  margin-left: 20px;
+  margin-top: 35px;
+  letter-spacing: 1px; 
+}
+.profile-level{
+  text-align: center;
+  color: #06468F;
+  margin-bottom: 50px;
+  margin-top: 30px;
+}
```
Comment (Fatmasiam) [inline Client/src/components/contanier/student/Profile/index.css]: new line

**Review by naremanhilles — APPROVED** [review]

File: Client/src/components/common/PickDay/index.css [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  font-family: Arial;
+  width: 180px;
+  color: darkblue;
+  height: 40px;
+  background-color: #d3d3d3;
+  border-radius: 5px;
+  border: none;
+  outline: none;
+  padding: 10px;
+  margin-left: 100px;
```
Comment (AsalaKM) [inline Client/src/components/common/PickDay/index.css]: reduce your code and replace it with:
`margin: 10px 0px 10px 100px`

File: Client/src/components/common/PickDay/index.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
```
Comment (AsalaKM) [inline Client/src/components/common/PickDay/index.js]: double quotation !!

File: Client/src/components/common/PickDay/index.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+import "./index.css";
+
+function Picker(props) {
+  const { option } = props 
+  return (
+    <select className="picker">
```
Comment (AsalaKM) [inline Client/src/components/common/PickDay/index.js]: double quotation !!!

File: Client/src/components/contanier/student/Apply/index.css [hunk 8]
```diff
@@ -0,0 +1,29 @@
+.apply-titel {
```
Comment (AsalaKM) [inline Client/src/components/contanier/student/Apply/index.css]: title
TYPO!

File: Client/src/components/contanier/student/Apply/index.js [hunk 9]
```diff
@@ -1,10 +1,30 @@
...
+import CourseCard from '../../../common/Course'
+import PickDay from '../../../common/PickDay'
+import Button from '../../../common/Button'
+import './index.css'
 
 export default class Apply extends Component {
+  state = {
+    course: {
+      id:0,
+      titel: 'Writing & Speaking',
```
Comment (AsalaKM) [inline Client/src/components/contanier/student/Apply/index.js]: title
TYPO !

File: Client/src/components/contanier/student/Apply/index.js [hunk 10]
```diff
@@ -1,10 +1,30 @@
...
   render() {
+    const { days, hours } = this.state
     return (
-      <div>
-        From apply
+      <div className='apply'>
+        <h1 className='apply-titel'>Apply</h1>
+        <CourseCard { ...this.state.course }/>
+        <hr className='apply-line' />
+        <h4 className='apply-tep'>Select from available options below</h4>
```
Comment (AsalaKM) [inline Client/src/components/contanier/student/Apply/index.js]: apply-tep ??
use clear class name

**Review by AsalaKM — CHANGES_REQUESTED** [review]

File: Client/src/components/common/PickDay/index.js [hunk 11]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+import "./index.css";
```
Comment (AsalaKM) [inline Client/src/components/common/PickDay/index.js]: double quotations ??!

**Review by AsalaKM — CHANGES_REQUESTED** [review]

**Review by Fatmasiam — APPROVED** [review]

File: Client/src/components/contanier/student/Apply/index.css [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+
+.apply-btn {
+  width: 80%;
+  height: 50px;
+  color: white;
+  background-color: #06468F;
+  border-radius: 5px;
+  outline: none;
+  font-size: 24px;
+  margin-top: 15px;
```
Comment (AsalaKM) [inline Client/src/components/contanier/student/Apply/index.css]:
```
margin-top: 15px;
margin-left: 35px;
```
Try to write it in one property 
```
margin
```

File: Client/src/components/common/PickDay/index.css [hunk 13]
```diff
@@ -0,0 +1,13 @@
+.picker {
+  font-size: 14px;
+  font-family: Arial;
+  width: 180px;
+  color: darkblue;
+  height: 40px;
+  background-color: #d3d3d3;
+  border-radius: 5px;
```
Comment (AsalaKM) [inline Client/src/components/common/PickDay/index.css:8]: why you need it ??
if the border is none !!

**Review by AsalaKM — CHANGES_REQUESTED** [review]

File: Client/src/components/common/PickDay/index.css (hunk 13 above)
Comment (AbdallahAmmar96) [reply Client/src/components/common/PickDay/index.css:8]: for the select list to be round, not sharp 
image

File: Client/src/components/contanier/student/Apply/index.css (hunk 12 above)
Comment (AbdallahAmmar96) [reply Client/src/components/contanier/student/Apply/index.css]: Done

File: Client/src/components/common/PickDay/index.css (hunk 13 above)
Comment (AsalaKM) [reply Client/src/components/common/PickDay/index.css:8]: okay!  I got it

**Review by AsalaKM — APPROVED** [review]

**Pull request merged**